src/docs/markdown/automatic-https.md

```diff
@@ -220,3 +222,84 @@ Before attempting any ACME transactions, Caddy will test the configured storage
 Caddy can obtain and manage wildcard certificates when it is configured to serve a site with a qualifying wildcard name. A site name qualifies for a wildcard if only its left-most domain label is a wildcard. For example, `*.example.com` qualifies, but these do not: `sub.*.example.com`, `foo*.example.com`, `*bar.example.com`, and `*.*.example.com`.
 
 To get a wildcard from Let's Encrypt, you simply need to enable the [DNS challenge](#dns-challenge) and use a wildcard domain in your config. We recommend using wildcards only when you have so many subdomains that you would encounter CA rate limits trying to obtain certificates for them all.
+
+
+
+## Examples
+
+Caddy implicitly uses the HTTPS port (default 443) for your [site addresses](/docs/conventions#network-addresses) that don't specify a port explicitly (which would disable automatic HTTPS). 
+
+Automatic HTTPS can be configured via the Caddyfile with the [`auto_https` global option](/docs/caddyfile/options), or via [per-server JSON configuration](/docs/json/apps/http/servers/automatic_https/). The Caddyfile option can be set to either of the following:
+- `disable_redirects` which disables the implicit HTTP->HTTPS redirect.
+- `off` which disables automatic HTTPS altogether, including the HTTP->HTTPS redirect and automatic enabling of TLS for sites that meet the requirements.
+
+| auto_https        | HTTP        | HTTPS       |
+|-------------------|-------------|-------------|
+| *default*         | redirects   | connects    |
+| disable_redirects | unreachable | connects    |
+| off               | connects    | unreachable |
+
+`unreachable` is the default without additional configuration when using the related `auto_https` setting.
+`connects` is the implicitly assigned port.
```

Comment:
This is also confusing. "Connects" is not a port. This should describe what connects means.

Reply:
This was my fault for only thinking of site addresses regarding ports(rather than Network Addresses), would it be more accurate to state that Caddy is implicitly adding the protocol scheme, which infers the port from `http_port` and `https_port` or their defaults?

The protocol scheme threw me off a bit, as while the http/https ports can be configured for Caddy to use different ports, the protocol schemes felt more explicit about what ports they'd refer to, but I guess they too match `http_port`/`https_port` and aren't related to the user connecting over `http/https` via their browser per se?

Reply:
Correct, if you specify a scheme, it uses the port configured for that scheme. By default this is, of course, 80 and 443. But in some circumstances you need to forward ports from the outside (I have to do this all the time), so you can change the ports for internal use.